<ui version="4.0" >
 <class>PluginLoaderGuiDesign</class>
 <widget class="QDialog" name="PluginLoaderGuiDesign" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plugin Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Available Plugins:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="Q3ListView" name="chooserView" >
      <property name="maximumSize" >
       <size>
        <width>200</width>
        <height>32767</height>
       </size>
      </property>
      <property name="allColumnsShowFocus" >
       <bool>false</bool>
      </property>
      <property name="resizeMode" >
       <enum>Q3ListView::AllColumns</enum>
      </property>
      <column>
       <property name="text" >
        <string>Load</string>
       </property>
      </column>
     </widget>
     <widget class="Q3GroupBox" name="settingsBox" >
      <property name="title" >
       <string>Settings</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="Q3WidgetStack" name="settingsStack" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="WStackPage" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>358</width>
            <height>364</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>458</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="focusPolicy" >
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3WidgetStack</class>
   <extends>Q3WidgetStack</extends>
   <header>q3widgetstack.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chooserView</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>PluginLoaderGuiDesign</receiver>
   <slot>saveValues()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooserView</sender>
   <signal>selectionChanged(Q3ListViewItem*)</signal>
   <receiver>PluginLoaderGuiDesign</receiver>
   <slot>pluginSelected()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
